### ISSUE TITLE
ARIA prop misspelled

### ISSUE THREAD

**Issue by loozhengyuan** [tailwindlabs/tailwindui-issues#399]
**Describe the bug**
I noticed that at least one of the ARIA prop is misspelled: `aria-has-popup` should be `aria-haspopup`.

**Additional context**
[ARIA Spec](https://www.w3.org/TR/wai-aria-1.1/#global_states)

**Comment by davidluhr** [comment]
@loozhengyuan Thank you for reporting this! We've fixed these attributes and the changes should be live on the site soon.
